staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -18,14 +18,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# generate-groups generates everything for a project with external types only, e.g. a project based
+# generate-groups generates everything for a project with external types, e.g. a project based
```

Comment:
there are *lots* of people calling this for [APIs which have no internal types defined](https://cs.github.com/?scopeName=All+repos&scope=&q=%2F%5C%2Fgenerate-groups.sh+%22%3Fall%2F). I'm pretty sure the conversion generator won't work for them, and I'm unsure about the other bits now included in `all`

Reply:
I question whether "all" is useful any more.  I could force it to be the historical meaning and just freeze it.  Or we could make it an error and tell people "replace 'all' with ..." locking in the historical meaning.

Prefs?

Reply:
The only opinions I have:
* if existing invocations are going to stop working, they should stop working in as clear and obvious a way as possible
* we should be able to recommend *something* to people who previously generated conversions for API types in two locations using this script

Other than that, I don't feel strongly about the shape this evolves into. @sttts might feel differently.